### DISCUSSION TITLE
Rocks, Ice cubes, and Icicles should push buttons, and activate switches.

### DISCUSSION THREAD

**Discussion by Alasdairbugs** [SuperTux/supertux#2055] [Ideas]
relevant to #1354 

This type of mechanic would offer a great increase in dynamic to gameplay. There may also be other objects that should interact with buttons. Also, it just makes sense. Things that have weight should be able to crush things that Tux normally would.

**Comment by tobbi** [comment]
Rocks do already push buttons.

**Comment by Alasdairbugs** [comment]
the other things should then as well

**Comment by celeste5000** [comment]
mrbomb, haywire, goldbomb should be able to activate switches too imo

**Comment by korbanpapa** [comment]
honestly, i think there should be an option for the item/badguy that's "trigger" or "trigger button" with a checkmark true/false. you can choose whatever option.
